src/applications/disability-benefits/526EZ/tests/schema/initialData.js

```diff
@@ -32,8 +32,8 @@ export default {
     {
       serviceBranch: 'Army',
       dateRange: {
-        from: '1990-02-02',
-        to: '2010-03-04'
+        // from: '1990-02-02',
```

Comment:
Can we put these back in for ease of testing later?

Reply:
Definitely.